CHANGES.rst

```diff
@@ -9,6 +9,10 @@
 
 - Enable test coverage reporting on coveralls.io and in tox.ini.
 
+- Host documentation at https://zopetestrunner.readthedocs.io
+
+- Remove undocumented support for the ``--pychecker`` option. See
+  `issue 63 <https://github.com/zopefoundation/zope.testrunner/issues/63>`_.
```

Comment:
I'm not sure we can call it "undocumented" when the `--help` output did document it a bit.  "Untested" might be a better word.

Reply:
OK.